MODULE_ALIAS_CRYPTO() was moved to crypto/algapi.h in kernel 6.4. See:
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=0c0edf6168ce1e02518ba44400b9269a13c3b9e6
---
 quickassist/qat/drivers/crypto/qat/qat_common/adf_ctl_drv.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/quickassist/qat/drivers/crypto/qat/qat_common/adf_ctl_drv.c b/quickassist/qat/drivers/crypto/qat/qat_common/adf_ctl_drv.c
index 8a33cc7..8e40e91 100644
--- a/quickassist/qat/drivers/crypto/qat/qat_common/adf_ctl_drv.c
+++ b/quickassist/qat/drivers/crypto/qat/qat_common/adf_ctl_drv.c
@@ -12,6 +12,7 @@
 #include <linux/device.h>
 #include <linux/notifier.h>
 #include <linux/reboot.h>
+#include <crypto/algapi.h>
 
 #include "adf_accel_devices.h"
 #include "adf_common_drv.h"
-- 

